osu.Game/Screens/Edit/Components/TimeInfoContainer.cs

```diff
@@ -36,26 +37,44 @@ private void load(OsuColour colours, OverlayColourProvider colourProvider)
                 bpm = new OsuSpriteText
                 {
                     Colour = colours.Orange1,
+                    Font = OsuFont.Torus.With(size: 14, weight: FontWeight.SemiBold, fixedWidth: true),
+                    Spacing = new Vector2(-1, 0),
+                    Position = new Vector2(0, 4),
+                    Anchor = Anchor.CentreRight,
+                    Origin = Anchor.TopRight,
+                },
+                progress = new OsuSpriteText
+                {
+                    Colour = colours.Purple1,
+                    Font = OsuFont.Torus.With(size: 14, weight: FontWeight.SemiBold, fixedWidth: true),
+                    Spacing = new Vector2(-1, 0),
                     Anchor = Anchor.CentreLeft,
-                    Font = OsuFont.Torus.With(size: 14, weight: FontWeight.SemiBold),
                     Position = new Vector2(2, 4),
                 }
             };
         }
 
         private double? lastBPM;
+        private double? lastProgress;
 
         protected override void Update()
         {
             base.Update();
 
             double newBPM = editorBeatmap.ControlPointInfo.TimingPointAt(editorClock.CurrentTime).BPM;
+            double newProgress = (int)(editorClock.CurrentTime / editorClock.TrackLength * 100);
```

Comment:
Not sure on this doing flooring, but it appears [to match `SongProgressInfo`](https://github.com/ppy/osu/blob/0e1b5802a9854342a03774d0a1d02a761a02b92f/osu.Game/Screens/Play/HUD/SongProgressInfo.cs#L148), and also does seem reach 100% at *some point*, so probably fine